caching/README.md

```diff
@@ -0,0 +1,159 @@
+# In-Memory Results Caching
+
+> Spice.ai OSS supports in-memory caching of query results.
+> Results caching can help improve performance for bursts of requests and for non-accelerated results such as refresh data returned [on zero results](https://docs.spiceai.org/data-accelerators/data-refresh#behavior-on-zero-results).
+>
+> [Spice.ai OSS Docs: Results Caching](https://docs.spiceai.org/features/caching)
+
+The quickstart will use [ TPC-H Benchmark Sample Data](https://github.com/spiceai/quickstarts/tree/trunk/tpc-h), but instead of using local acceleration, we will leverage in-memory caching to boost query performance.
```

Comment:
```suggestion
The quickstart will use [TPC-H Benchmark Sample Data](https://github.com/spiceai/quickstarts/tree/trunk/tpc-h), but instead of using local acceleration, we will leverage in-memory caching to boost query performance.
```